pipelinewise/cli/pipelinewise.py

```diff
@@ -1621,6 +1655,141 @@ def encrypt_string(self):
+                self.tap_run_log_file = os.path.join(
+                    log_dir, f'{target_id}-{tap_id}-{current_time}.partialsync.log'
+                )
+
+                # Create parameters as NamedTuples
+                tap_params = TapParams(
+                    tap_id=tap_id,
+                    type=tap_type,
+                    bin=self.tap_bin,
+                    python_bin=self.tap_python_bin,
+                    config=tap_config,
+                    properties=tap_properties,
+                    state=tap_state,
+                )
+
+                target_params = TargetParams(
+                    target_id=target_id,
+                    type=target_type,
+                    bin=self.target_bin,
+                    python_bin=self.target_python_bin,
+                    config=cons_target_config,
+                )
+
+                transform_params = TransformParams(
+                    bin=self.transform_field_bin,
+                    config=tap_transformation,
+                    python_bin=self.transform_field_python_bin,
+                    tap_id=tap_id,
+                    target_id=target_id,
+                )
+
+                self.run_tap_partialsync(
+                    tap=tap_params, target=target_params, transform=transform_params,
+                )
+
+        except pidfile.AlreadyRunningError:
+            self.logger.error('Another instance of the tap is already running.')
+            sys.exit(1)
+        # Delete temp file if there is any
+        except commands.RunCommandException as exc:
+            self.logger.exception(exc)
+            self.send_alert(message=f'Failed to sync tables in {tap_id} tap', exc=exc)
+            sys.exit(1)
+        except Exception as exc:
+            self.send_alert(message=f'Failed to sync tables in {tap_id} tap', exc=exc)
+            raise exc
+        finally:
+            utils.silentremove(cons_target_config)
+
+
+    def _check_supporting_tap_and_target_for_partial_sync(self):
+        tap_type = self.tap['type']
+        tap_id = self.tap['id']
+        target_type = self.target['type']
+        target_id = self.target['id']
+
+        if target_type != 'target-snowflake':
+            print(f'Error! target "{target_id} (type: {target_type})" is not supported for partial sync!')
+            sys.exit(1)
+
+        if tap_type not in ['tap-mysql', 'tap-postgres']:
+            print(f'Error! tap "{tap_id} (type: {tap_type})" is not supported for partial sync!')
+            sys.exit(1)
+
+    def _check_if_complete_tap_configuration(self, fastsync_bin, tap_type, target_type):
+        # Tap exists but configuration not completed
+        if not os.path.isfile(fastsync_bin):
+            self.logger.error(
+                'Table sync function is not implemented from %s datasources to %s type of targets',
+                tap_type,
+                target_type
+            )
+            sys.exit(1)
```

Comment:
- [ ] Fix it!

Reply:
These `sys.exit(1)`'s are not something new and they existed there. since it is not in this PR scope and also I worry if it causes some tests/functions break, I prefer not to touch them